LIPs/LIP-25.md

```diff
@@ -0,0 +1,237 @@
+---
+lip: 25
+title: Extensible Governance Contract
+author: Nico Vergauwen (@kyriediculous) <nico@livepeer.org>
+type: Meta
+status: Draft
+created: 2020-06-09
+requires: n/a
+---
```

Comment:
The `discussions-to` field needs to be specified. See [LIP-1](https://github.com/livepeer/LIPs/blob/master/LIPs/LIP-1.md#lip-header-preamble) for more details.

Reply:
fixed in 1afd126